apply/default_applier.go

```diff
@@ -259,6 +259,8 @@ func (c *DefaultApplier) diff() (todoList []ActionName, err error) {
 	}
 	todoList = append(todoList, Guest)
 	todoList = append(todoList, UnMountImage)
+	todoList = append(todoList, PluginDump)
+	todoList = append(todoList, PluginPhasePostInstallRun)
```

Comment:
plugin post install already in the currentcluster == nil block.

Reply:
This func only works in installing phrase. if I want to apply this feature after installing k8s, it will not work.

Reply:
Some plugin exec when the upgrade may occur some error,  like yum install some already exist tools.  But if not add this the new node will not exec the plugin command...